pkg/securitycontext/provider.go

```diff
@@ -95,3 +96,53 @@ func makeCapabilites(capAdd []api.Capability, capDrop []api.Capability) ([]strin
 	}
 	return addCaps, dropCaps
 }
+
+func determineEffectiveSecurityContext(pod *api.Pod, container *api.Container) *api.SecurityContext {
```

Comment:
I am not sure I follow.  If we assert that non-specified values in container.SC mean "do the right thing" and always have, why would a non-nil CSC with no values in it take precedence over PSC with values?